src/main/java/seedu/duke/util/Parser.java

```diff
@@ -399,7 +387,7 @@ private Command prepareAddEvent(String userMessage) throws SystemException {
      * @return A string of converted content input.
      * @throws StringIndexOutOfBoundsException if an error occurs.
      */
-    public String inputContent() throws StringIndexOutOfBoundsException {
+    public static String inputContent() throws StringIndexOutOfBoundsException {
```

Comment:
Catch the StringIndexOutOfBoundsException within the function and return null instead.

Reply:
I made it loop till there is content in the input.